yarn.lock

```diff
@@ -4924,14 +4924,22 @@
   dependencies:
```

Comment:
I think this changeset in `yarn.lock` is indicative of running `yarn` in a sub package and not in the root of the repo

Reply:
Am I not supposed to do that?

Reply:
I think it is shunned upon

Reply:
Care to elaborate?